doc/source/user_guide/meshing_workflow/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _ref_meshing_workflow:
+
+================
+Meshing Workflow
+================
+Fluent meshing guided workflows are designed to make it easier to generate a
+volume mesh starting from CAD geometries. You can use PyFluent to access the guided workflows.
+
+See :ref:`ref_user_guide_meshing_workflows` for the earlier interface that appears
+in journals or :ref:`ref_user_guide_new_meshing_workflows` for the improved Pythonic interface.
```

Comment:
```suggestion
Workflows are divided into the new :ref:`ref_user_guide_new_meshing_workflows` and :ref:`ref_user_guide_meshing_workflows` which align with the journaled syntax.
```

Reply:
N.b. I added a comma here to make it clear that only the latter is aligned with journaling.